src/sentry/issues/grouptype.py

```diff
@@ -18,6 +28,22 @@ class GroupCategory(Enum):
 _group_type_registry: Dict[int, Type[GroupType]] = {}
 _slug_lookup: Dict[str, Type[GroupType]] = {}
 _category_lookup: Dict[int, Set[int]] = defaultdict(set)
+DEFAULT_IGNORE_LIMIT: int = 3
+DEFAULT_EXPIRY_TIME: int = 60 * 60 * 24
+
+
+@dataclass(frozen=True)
+class NoiseConfig:
+    ignore_limit: int = DEFAULT_IGNORE_LIMIT
```

Comment:
Should this be removed out of `NoiseConfig`, since it already exists on `GroupType`?

I'm not sure we need different ignore limits per rollout type, although I suppose it could be useful for testing different ignore limits on Sentry first?

My main thought here was that for EA/LA/GA that we'd have rollout rates.

Reply:
Next PR was going to move it out of `GroupType`. I guess I'd need to think about rollout procedure a bit more. Do we want to generally have different policies/rollout rates running in parallel before unifying them as a GA value? Or is it more LA->EA->GA in discrete stages?

Reply:
I feel like these specific noise settings are probably separate to the EA/LA/GA settings. I don't know if we'd want to have separate settings for customers in different stages, it's more of a rollout thing, and we're applying the same set of settings to a widening group of customers.

I can see us potentially wanting to try out different settings on different customers are some point, but it might be better to wait until we have a need for that explicitly. Like maybe we want to trial a higher ignore limit on Sentry or something. But I think that should be separate to our rollout strategy.

Reply:
Alright...this is starting to make some sense. Might sync up later with more Q's

Reply:
Actually, I suppose one case is to have the ignore limit just disabled for a new group type in LA, which is just sentry orgs, then EA/GA have a standard level.

If we did want to do this, maybe it makes sense to have a default ignore limit set on the group type, and have that be able to be overridden?

Reply:
> Alright...this is starting to make some sense. Might sync up later with more Q's

No worries, ping me on slack or set up a google meet if that's helpful